parlai/core/agents.py

```diff
@@ -286,54 +293,75 @@ def create_agent_from_opt_file(opt: Opt):
     """
     model_file = opt['model_file']
     optfile = model_file + '.opt'
-    if os.path.isfile(optfile):
```

Comment:
to clarify - the only functional change here was moving this block out of the conditional statement right?

Reply:
yes and some renaming. i changed "new_opt" to "opt_from_file" because I thought the naming was confusing